ScoutSuite/frontend/src/pages/Resources/ResourcePartialWrapper/style.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/variables';
```

Comment:
same for this file, we should just keep the style we need rather than copying everything.

Reply:
Okay, yes, I will take note of this but make the change I my next PR for resources (probably tomorrow or Friday). I think you are right, we should have wrapper for this.